doc/iMac_in_the_lab.md

```diff
@@ -1,6 +1,6 @@
 There are currently (update: July 2024) 4 iMacs in the lab.
-If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?gid=0#gid=0).
+If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?usp=sharing).
 
 It is good practice to create your own account on the computer in order not to install on the same account a different version of Matlab Tools.
 
-All iMacs have an **admin** account, password : **CPP**.
+All iMacs have an **admin** account. Ask a lab member for the password.

```

Comment:
What do you think about creating a column to the sheet file mentioned above with the pss? It won't be visible by external people. LMK what you think and I will do it.

Reply:
Yes it is done. I added passwords that I know, feel free to add others.